optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -70,6 +76,8 @@
     @XmlElement(name = "scoreDrlFile")
     protected List<File> scoreDrlFileList = null;
 
+    protected Boolean compileDroolsAlphaNetwork = null;
```

Comment:
Naming: maybe droolsAlphaNetworkCompilationEnabled ?

Reply:
reasoning: start with drools for context + easier to find if you were looking for something like this
end with Enabled to make it clear it's a switch and what true/false actually does (= turn it on/off)